Numsense.UnitTests/RussianExamples.fs

```diff
@@ -0,0 +1,244 @@
+[<InlineData(                                         "тридцать",         30)>]
+[<InlineData(                                  "тридцать четыре",         34)>]
+[<InlineData(                                  "тридцать восемь",         38)>]
+[<InlineData(                                            "сорок",         40)>]
+[<InlineData(                                       "сорок один",         41)>]
+[<InlineData(                                     "сорок четыре",         44)>]
+[<InlineData(                                        "пятьдесят",         50)>]
+[<InlineData(                                   "пятьдесят семь",         57)>]
+[<InlineData(                                       "шестьдесят",         60)>]
+[<InlineData(                                  "шестьдесят пять",         65)>]
+[<InlineData(                                        "семьдесят",         70)>]
+[<InlineData(                                   "семьдесят семь",         77)>]
+[<InlineData(                                 "семьдесят девять",         79)>]
+[<InlineData(                                      "восемьдесят",         80)>]
+[<InlineData(                                "восемьдесят шесть",         86)>]
+[<InlineData(                                        "девяносто",         90)>]
+[<InlineData(                                    "девяносто три",         93)>]
+[<InlineData(                                              "сто",        100)>]
+[<InlineData(                                         "сто один",        101)>]
+[<InlineData(                                           "двести",        200)>]
+[<InlineData(                             "двести тридцать семь",        237)>]
+[<InlineData(                            "триста семьдесят пять",        375)>]
+[<InlineData(                              "четыреста девяносто",        490)>]
+[<InlineData(                           "пятьсот шестьдесят три",        563)>]
+[<InlineData(                            "шестьсот восемнадцать",        618)>]
+[<InlineData(                           "семьсот семьдесят семь",        777)>]
+[<InlineData(                                   "восемьсот пять",        805)>]
+[<InlineData(                       "девятьсот пятьдесят девять",        959)>]
+[<InlineData(                                           "тысяча",       1000)>]
+[<InlineData(                                      "одна тысяча",       1000)>]
+[<InlineData(                                      "тысяча один",       1001)>]
+[<InlineData(                                  "одна тысяча два",       1002)>]
+[<InlineData(                    "одна тысяча шестьдесят четыре",       1064)>]
+[<InlineData(                                       "две тысячи",       2000)>]
+[<InlineData(                                       "три тысячи",       3000)>]
+[<InlineData(                                    "четыре тысячи",       4000)>]
+[<InlineData(                                      "пять тысячи",       5000)>]

```

Comment:
Oh, then all ok. Yes, it's even better if user can make some mistakes. I thought that's a typo)